### DISCUSSION TITLE
MNIST Duet Server syft MemoryStore delitem error

### DISCUSSION THREAD

**Discussion by cbs-rschreijen** [OpenMined/courses#301] [Course: Foundations of Private Computation] [answered]
Hi, I managed to successfully run the duet MNIST example with a local loopback duet server/client. Everything works fine, including full training and inference.

I just noticed a really large nr. of errors in the server notebook when communicating with the client. They all look like this, but do not seem to affect the correct functionality:

[2021-03-25T11:12:53.642171+0100][CRITICAL][logger]][91909] <class 'syft.core.store.store_memory.MemoryStore'> __delitem__ error <UID: 42f91d657fe24ebe8fb879d2a1122b96>.

**Comment by znreza** [comment, marked as answer]
Hi @cbs-rschreijen sometimes occasionally you may see this types of errors in the server notebook, which may or may not arise due to some internal error in the duet server. This particular error arises when your notebook at some point is trying to delete an item of the given UID but the item is not present in the store. I believe if you run the cells again step by step, you will not see this error next time. You can find more details of this specific error in [here](https://pysyft.readthedocs.io/en/latest/_modules/syft/core/store/store_memory.html) and [here](https://pysyft.readthedocs.io/en/latest/api/syft.core.store.html#syft.core.store.MemoryStore.delete).
